examples/rust_control_interface/src/main.rs

```diff
@@ -80,6 +80,7 @@ fn create_request_to_add_new_workload() -> ToAnkaios {
                 dependencies: Some(Dependencies {
                     dependencies: HashMap::new(),
                 }),
+                files: None,
                 configs: None,
                 control_interface_access: None,
```

Comment:
We could use a default here to reduce the effort next time:
```suggestion
                ..Default::default()
```

Reply:
Fixed in separate commit because of testing on-the-fly: [a1531ac](https://github.com/eclipse-ankaios/ankaios/pull/426/commits/a1531ac09e527d5661ab11b72b4c5e9426022f18)